templates/nro/.circleci/config.yml.tmpl

```diff
@@ -525,6 +552,26 @@ workflows:
 {{- end }}
 
 {{- if isTrue .Env.MAKE_MASTER }}
+  rollback:
+    when: << pipeline.parameters.rollback >>
+    jobs:
+      - hold-staging:
```

Comment:
Do we need this manual approval step? The workflow will already be triggered manually if I understand correctly.

Reply:
Removed :+1: